news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
@badfiles although in this case, the situation may also be somewhat different - AAT authors may or may not approve with the way there template is used.  I'm querying as a contributor to MantisBT whether the inclusion by Mantishub LLC of commercial code in code that I have an interest in is allowed. 

The drupal project ( https://www.drupal.org/about/licensing#q1 ) is licensed under the GPL. They state:

Does the license cover just PHP, or everything?

> 
> We require that all files (PHP, JavaScript, images, Flash, etc.) hosted on Drupal.org be under the GPL. If it's in Git, then it is under the same license (GPL version 2 or later). That way there is no confusion about what license a given file is under.
> 
> Who holds the copyright for Drupal?
> 
> Drupal and contributed modules and themes include code written by thousands of different people. All Drupal contributors retain copyright on their code, but agree to release it under the same license as Drupal. If you are unable or unwilling to contribute a patch under the GPL version 2 or later, do not submit a patch.

This would seem to support my analysis - so from a drupal standpoint - and I suspect as a larger project they have probably checked out the situation properly, the patch that includes the AAT should be licensed under the GPL or not submitted - so the question here may be - is the AAT available under GPL2?